src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -2,43 +2,406 @@
+            };
+
+            return Unsafe.AsRef<Vector128<short>>(pResult);
+        }
+
+        public static unsafe Vector128<int> Create(int value)
+        {
+            var pResult = stackalloc int[4]
+            {
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<int>>(pResult);
+        }
+
+        public static unsafe Vector128<long> Create(long value)
+        {
+            var pResult = stackalloc long[2]
+            {
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<long>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector128<sbyte> Create(sbyte value)
+        {
+            var pResult = stackalloc sbyte[16]
+            {
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<sbyte>>(pResult);
+        }
+
+        public static unsafe Vector128<float> Create(float value)
+        {
+            var pResult = stackalloc float[4]
+            {
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<float>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector128<ushort> Create(ushort value)
+        {
+            var pResult = stackalloc ushort[8]
+            {
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<ushort>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector128<uint> Create(uint value)
+        {
+            var pResult = stackalloc uint[4]
+            {
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<uint>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector128<ulong> Create(ulong value)
+        {
+            var pResult = stackalloc ulong[2]
+            {
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector128<ulong>>(pResult);
+        }
+
+        public static unsafe Vector128<byte> Create(byte e0, byte e1, byte e2, byte e3, byte e4, byte e5, byte e6, byte e7, byte e8, byte e9, byte e10, byte e11, byte e12, byte e13, byte e14, byte e15)
```

Comment:
The input parameters here are ordered from least to greatest (e0, e1, ..., e15); while the current x86 `SetVector` APIs order them from greatest to least (e15, e14, ..., e0).

I changed it here because it better matches the .NET convention and the normal expectation. The native x86 intrinsics also expose `_mm_setr` methods that match this "expected" ordering.

Reply:
We reviewed this today and determined the new ordering is better/correct